vspace/src/main/java/edu/asu/diging/vspace/web/staff/DeleteLinkController.java

```diff
@@ -41,4 +45,10 @@ public ResponseEntity<String> deleteExternalLink(@PathVariable("id") String spac
         externalLinkManager.deleteLink(linkId);
         return new ResponseEntity<String>(HttpStatus.OK);
     }
+    
+    @RequestMapping(value = "/staff/module/{id}/slide/{slideId}/externallink/{linkId}", method = RequestMethod.DELETE)
+    public ResponseEntity<String> deleteSlideExternalLink(@PathVariable("id") String moduleId, @PathVariable("slideId") String slideId, @PathVariable("linkId") String linkId) {
```

Comment:
this should not be in this controller. the other methods are for deleting links from a space, this one is for deleting links from a slide.

Reply:
?

Reply:
so this is my only controller for deleting links throughout the project. It just so happens that the only other place with links is a space. `DeleteSlideController` handles deleting slides so I don't want to throw in a method that deletes links as well. I could create a new controller just for deleting links from slides but that seems excessive.